server/controllers/notification.js

```diff
@@ -35,20 +35,52 @@ exports.getUnread = expressAsyncHandler(async (req, res, next) => {
   res.status(200).json({
     success: {
       notifications: notifications,
-    }
+    },
   });
 });
 
-exports.markAsread = expressAsyncHandler(async (req, res, next) => {
+exports.markAllAsRead = expressAsyncHandler(async (req, res, next) => {
   const user = await User.findById(req.user.id);
 
   if (!user) {
     res.status(401).send("Unathenticated");
     throw new Error("Not authorized");
   }
 
+  notification.read = true;
+  notification.read_date = new Date().toString();
```

Comment:
What is the plan with read_date? For filtering notifcations by when they read them?

Reply:
Also, please use camelCase